app/streaming/input/gamepad.cpp

```diff
@@ -900,6 +900,17 @@ void SdlInputHandler::setControllerLED(uint16_t controllerNumber, uint8_t r, uin
 #endif
 }
 
+void SdlInputHandler::setAdaptiveTriggers(uint16_t controllerNumber, DualSenseOutputReport *report){
+
+#if SDL_VERSION_ATLEAST(2, 0, 16)
+    if (controllerNumber <= MAX_GAMEPADS && m_GamepadState[controllerNumber].controller != nullptr) {
+        SDL_GameControllerSendEffect(m_GamepadState[controllerNumber].controller, report, sizeof(*report));
```

Comment:
We probably need to check if the gamepad is a PS5 controller before we send the output report to it.

Reply:
Thanks, makes sense. I've added a check for it